apps/site/assets/ts/schedule/components/line-diagram/StopCard.tsx

```diff
@@ -78,7 +78,12 @@ const StopCard = (props: StopCardProps): ReactElement<HTMLElement> => {
   const showPrediction = hasLivePredictions && !isDestination;
   const showDiversion =
     diversionAlert && !(hasLivePredictions && isDestination);
-
+  const showDepartures = (): string => {
```

Comment:
suggestion (non-blocking): Personally, this function name sounds like calling it would trigger an action. I wonder if something in the ballpark of `buttonLabel` might be a closer description? (I'm terrible at coming up with names, and that's also pretty generic sounding, so take it in whatever direction you want.)

Reply:
Sorry, one additional suggestion: I think this logic could warrant a super quick test or two.